pypeman/contrib/http.py

```diff
@@ -12,6 +13,9 @@
 
 logger = logging.getLogger(__name__)
 
+str_named_param_regex = re.compile(r"%\((?P<keyval>.*?)\)[r|s|d|]")
```

Comment:
you could add comments with examples what is expected to be matched by this regexps

Reply:
shouldn't `<keyval>.*` be something like [a-zA-Z0-9_\.]* ? or are there really complex expressions containing other characters?

`*?` instead of '*' makes only sense if a `)` character were allowed for `keyval`

Reply:
or just use `[^)]*` or `[^\)]*` (check if `)` must be scaped